### PR TITLE
Update information about kubernetes metrics

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update information about kubernetes metrics 

- Change scheduler metrics according to changes added in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/795
- Change cAdvisor and Node exporter  metrics according to metrics mentioned in values.yaml for Sumologic Kuberentes Helm Chart
- Change information about aggregated metrics
- Change list of kube-state-metrics metrics recommended to collect, use metrics from https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/scraped-metrics.md#metrics

Issue number: https://jira.kumoroku.com/jira/browse/SUMO-213876

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/metrics/kubernetes-metrics.md [hunk 1]
```diff
@@ -6,122 +6,120 @@ description: This page lists the Kubernetes metrics that are collected when you
...
-* Pods
-    * kube_pod_container_info
-    * kube_pod_container_resource_requests
-    * kube_pod_container_resource_limits
-    * kube_pod_container_status_ready
-    * kube_pod_container_status_terminated_reason
-    * kube_pod_container_status_waiting_reason
-    * kube_pod_status_phase
+This section lists recommended metrics to collect. For information about all kube-state-metrics, and metric descriptions, 
+see [kube-state-metrics docs](https://github.com/kubernetes/kube-state-metrics/tree/master/docs)
```
Comment (sumo-drosiek) [inline docs/metrics/kubernetes-metrics.md:25]: is pointing to `master` safe?

Comment (kkujawa-sumo) [reply docs/metrics/kubernetes-metrics.md:25]: I can change the link to point to release branch but we will forget probably to change it https://github.com/kubernetes/kube-state-metrics/tree/release-2.6/docs 🤷

Comment (kkujawa-sumo) [reply docs/metrics/kubernetes-metrics.md]: I changed the link, we will see in the future if we are able to update it ;)

Comment (sumo-drosiek) [reply docs/metrics/kubernetes-metrics.md]: Agree, unfortunately there is no good solution, both of them can end with incorrect information :see_no_evil:

File: docs/metrics/kubernetes-metrics.md [hunk 2]
```diff
@@ -6,122 +6,120 @@ description: This page lists the Kubernetes metrics that are collected when you
...
+Information about filtering and relabeling metrics, and how to send custom Prometheus metrics to Sumo Logic can be found in
+[Collecting Application Metrics guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/collecting-application-metrics.md).
 
 ## Kube state metrics (kube-state-metrics)
 
-kube-state-metrics is a service that listens to the Kubernetes API server and generates metrics about the state of the objects, including deployments, nodes, and pods.
+[kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) is a service that listens to the Kubernetes API server and generates metrics about the state of the objects, including deployments, nodes, and pods.
 
-You can deploy kube-state-metrics using Helm. The Sumo Logic helm chart has a dependency to install it. To avoid duplication, use a single instance of FluentD to collect the metrics.  
+You can deploy kube-state-metrics using [The Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
```
Comment (kimsauce) [inline docs/metrics/kubernetes-metrics.md:18]:
```suggestion
You can deploy kube-state-metrics using the [Sumo Logic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection).
```

File: docs/metrics/kubernetes-metrics.md [hunk 3]
```diff
@@ -6,122 +6,120 @@ description: This page lists the Kubernetes metrics that are collected when you
...
 
-### View kube-scheduler metrics 
-
-Install [Prometheus Operator](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/main/deploy#step-2-configure-prometheus) on your cluster in the prometheus namespace.
-
-Run this command to start a proxy to the Kubernetes API server: 
+### View kube-scheduler metrics
 
-`kubectl proxy`
+To see kube-scheduler metrics run the curl command from a master node, using the appropriate certificates to pass the authentication process:
```
Comment (kimsauce) [inline docs/metrics/kubernetes-metrics.md:111]:
```suggestion
To see kube-scheduler metrics, run the curl command from a master node, using the appropriate certificates to pass the authentication process:
```

File: docs/metrics/kubernetes-metrics.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/metrics/kubernetes-metrics.md]: Changed

File: docs/metrics/kubernetes-metrics.md (hunk 3 above)
Comment (kkujawa-sumo) [reply docs/metrics/kubernetes-metrics.md]: Changed

**Review by kimsauce — APPROVED** [review]

**Pull request merged**